composer.json

```diff
@@ -55,7 +55,8 @@
         "psr/cache-implementation": "To use metadata caching.",
         "symfony/cache": "To have metadata caching when using Symfony integration.",
         "symfony/config": "To load XML configuration files.",
-        "symfony/twig-bundle": "To use the Swagger UI integration."
+        "symfony/twig-bundle": "To use the Swagger UI integration.",
+        "swagger-api/swagger-ui": "^2.2"
```

Comment:
If we use `api-platform/core` in a Symfony project and if we don't require this suggested package, the `SwaggerUiListener` class and the `SwaggerUiAction` class will continue to do their job and a broken page will be displayed.